explainers/subresource-loading.md

```diff
@@ -0,0 +1,101 @@
+# Explainer: Subresource loading with Web Bundles
+
+Last updated: March 2020
+
+We propose a new approach to load a large number of resources efficiently using
+a format that allows multiple resources to be bundled, e.g.
+[Web Bundles](https://web.dev/web-bundles/).
+
+## Backgrounds
+
+- Loading many unbundled resources is still slower in 2020. We concluded that
+  [bundling was necessary in 2018](https://v8.dev/features/modules#bundle), and
+  our latest local measurement still suggests that.
+
+- Caching stories with JS bundlers (e.g. webpack) is tough. They are pretty good
+  tools but configuring them to work in an optimal way is tough, and sometimes
+  also conflicts with new requirements like dynamic bundling (e.g. small edit
```

Comment:
```suggestion
  they're also incompatible with new requirements like dynamic bundling (e.g. small edit
```

And please link "dynamic bundling" to a description.

Reply:
Thanks. Done.

Reply:
Ah, I didn't add link "dynamic bundling". I think it's still an early idea, and we don't have a good description of that in public,

Reply:
This one, maybe?

https://github.com/azukaru/progressive-fetching/blob/master/docs/dynamic-bundling/index.md

Reply:
Thanks! Let me add a link to that. I think that is okay for us.